doc/common_tools_writing_lints.md

```diff
@@ -83,6 +105,32 @@ A list of defined paths for Clippy can be found in [paths.rs][paths]
 
 We access lang items through the type context `tcx`. `tcx` is of type [`TyCtxt`][TyCtxt] and is defined in the `rustc_middle` crate.
 
+# Checking if a type defines a specific method
+
+To check if our type defines a method called `some_method`:
```

Comment:
r? @phansch 

Was this what you had in mind for this part of the documentation?

Reply:
Yup, LGTM!